.github/workflows/integration.yml

```diff
@@ -271,6 +271,26 @@ jobs:
       - name: build proposals tests
         run: yarn build
         working-directory: a3p-integration
+      - name: zip up all core proposals
+        run: |
+          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
+            parent_dir=$(basename "$(dirname "$dir")")
+
+            sanitized_parent_dir=${parent_dir//:/-}
+
+            zip_file="${sanitized_parent_dir}.zip"
+
+            zip -r "$zip_file" "$dir"
+
+            echo "Zipped $dir into $zip_file"
+          done
+      - name: archive core proposals
+        uses: actions/upload-artifact@v4
```

Comment:
According to https://github.com/actions/upload-artifact, `retention-days` will default to repository setting, which currently is set at 90 days